lib/src/models/crypto.ts

```diff
@@ -27,3 +29,20 @@ export interface JWKInterface {
     alg: string;
     n: string;
 }
+
+export interface CryptoUtils {
+    base64URLEncode(value: Buffer | string): string;
+    getCodeVerifier(env :string): string;
```

Comment:
We can combine `getCodeVerifier` and `getCodeChallenge`, right?

Reply:
Cannot combine these two as we are setting the code verifier to the data layer